bin/supply.ps1

```diff
@@ -3,37 +3,30 @@ echo "-----> DatadogBuildpack/supply"
 $BIN_DIR=Split-Path -Path $PSCOMMANDPATH
 $ROOT_DIR=Split-Path -Path $BIN_DIR
 $BUILD_DIR=$ARGS[0]
-$CACHE_DIR=$ARGS[1]
-$ENV_DIR=$ARGS[2]
+# $CACHE_DIR=$ARGS[1]
+# $ENV_DIR=$ARGS[2]
 
-echo "       Installing Datadog Agent"
+echo "       Extracting Datadog Agent"
 
 
 New-Item -Path "$BUILD_DIR\datadog\scripts" -ItemType Directory
-New-Item -Path "$BUILD_DIR\datadog\extracted-agent" -ItemType Directory
 New-Item -Path "$BUILD_DIR\.profile.d" -ItemType Directory
+New-Item -Path "$BUILD_DIR\datadog/AppData" -ItemType Directory
 
 # Extract agent
-Start-Process "MSIEXEC" -ArgumentList "/a $ROOT_DIR\lib\datadog-agent.msi /qn TARGETDIR=$BUILD_DIR\datadog\extracted-agent" -Wait -NoNewWindow
-Move-Item -Path "$BUILD_DIR\datadog\extracted-agent\Datadog\Datadog Agent" -Destination "$BUILD_DIR\datadog\"
-Move-Item -Path "$BUILD_DIR\datadog\extracted-agent\CommonAppData\Datadog" -Destination "$BUILD_DIR\datadog\AppData"
-Remove-Item -Force -Recurse -Path "$BUILD_DIR\datadog\extracted-agent"
+Expand-Archive -LiteralPath $ROOT_DIR\lib\cf-windows-7.17.0-buildpack.zip -DestinationPath $BUILD_DIR\datadog
```

Comment:
Can we have a versionless zip name ?

Reply:
Yeah for sure. This part isn't final yet until the build script is updated to pull the artifact from somewhere. Waiting on an agent PR before I can add that 😄